### PR TITLE
Allow offsetting tracking lights

### PR DESCRIPTION
**Pull request by jrb0001**

### CODE REVIEW

File: rts/Sim/Projectiles/Projectile.cpp [hunk 1]
```diff
@@ -212,7 +212,7 @@ bool CProjectile::GetMemberInfo(SExpGenSpawnableMemberInfo& memberInfo)
 	if (CExpGenSpawnable::GetMemberInfo(memberInfo))
 		return true;
 
-	CHECK_MEMBER_INFO_FLOAT3(CProjectile, dir)
+	//CHECK_MEMBER_INFO_FLOAT3(CProjectile, dir)
```
Comment (jrb0001) [inline rts/Sim/Projectiles/Projectile.cpp]: I don't know spring good enough to find the best way to solve this. Please let me know how you would do it.

**Comment by ashdnazg** [comment]
Can you please give a concise explanation of what you are trying to do?

**Comment by ashdnazg** [comment]
In general I recommend discussing changes in the forums/#sy before pull requests, especially when you "don't know spring good enough".

**Comment by rtri** [comment]
From the code, it looks like the intention is to offset (=translate) tracking lights within a tracked object's frame but this might not be the best way to get there.

**Comment by ashdnazg** [comment]
I'm still not sure where the translation is supposed to come from

**Comment by rtri** [comment]
`light.GetPosition()` probably, interpreted as a vector relative to `light.GetTrackPosition()`

**Comment by jrb0001** [comment]
Yes, the idea is to translate and rotate the lights by their own position and rotation relative to the position and rotation of the unit or projectile which they are tracking.

**Pull request closed without merging**